src/components/UserInfo/UserInfo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+type Props = {
+  name: string | undefined,
+  email: string | undefined,
```

Comment:
```suggestion
  name?: string,
  email?: string,
```

It would have the same effect

Reply:
👍👍